src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Color color;
+    private final int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball {" + "color='" + color + '\'' + ", number=" + number + '\'' + '}';
```

Comment:
[use method name() to get string representation for enum](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)
```suggestion
        return "Ball {" + "color='" + color.name() + '\'' + ", number=" + number + '\'' + '}';
```